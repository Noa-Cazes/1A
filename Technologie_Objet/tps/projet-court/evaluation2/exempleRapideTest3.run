java allumettes.Partie Xavier@humain Ordinateur@rapide <<EOF
1
1
2
EOF

