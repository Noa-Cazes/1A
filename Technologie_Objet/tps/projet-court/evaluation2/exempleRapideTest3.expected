
Nombre d'allumettes restantes : 13
Au tour de Xavier.
Combien prenez-vous d'allumettes ? Xavier prend 1 allumette.

Nombre d'allumettes restantes : 12
Au tour de Ordinateur.
Ordinateur prend 3 allumettes.

Nombre d'allumettes restantes : 9
Au tour de Xavier.
Combien prenez-vous d'allumettes ? Xavier prend 1 allumette.

Nombre d'allumettes restantes : 8
Au tour de Ordinateur.
Ordinateur prend 3 allumettes.

Nombre d'allumettes restantes : 5
Au tour de Xavier.
Combien prenez-vous d'allumettes ? Xavier prend 2 allumettes.

Nombre d'allumettes restantes : 3
Au tour de Ordinateur.
Ordinateur prend 3 allumettes.
Ordinateur a perdu !
Xavier a gagnÃ© !
